presto-main/src/main/java/com/facebook/presto/server/ServerMainModule.java

```diff
@@ -402,8 +380,7 @@ protected void setup(Binder binder)
         binder.bind(Metadata.class).to(MetadataManager.class).in(Scopes.SINGLETON);
 
         // statistics calculator
-        binder.bind(StatsCalculator.class).annotatedWith(SelectingStatsCalculator.Old.class).to(CoefficientBasedStatsCalculator.class).in(Scopes.SINGLETON);
-        binder.bind(StatsCalculator.class).to(SelectingStatsCalculator.class).in(Scopes.SINGLETON);
```

Comment:
What about cost calculator (next 5 lines)? 
Apparently it doesn't make sense to treat them separate.

Reply:
@electrum will you bind cost calculator only on the coordinator as well?